src/system/template-path-hints.md

```diff
@@ -32,6 +32,10 @@ Before using template path hints, add your IP address to the [whitelist]({% lin
 
     - To activate template path hints for the store, set **Enabled Template Path Hints for Storefront** to `Yes`.
 
+    - To enable template path hints for the store with only passing params in url, set **Enable Hints for Storefront with URL Parameter** to `Yes`.
```

Comment:
What does `hints for the store with only passing params in url` mean? Is this a way of further defining the previous item so that hints are only displayed for that items that have passed parameters? Does this help the developer focus on only those items that have different coding to review/debug?

Reply:
@leslietilling that means template hints are showed only when passing params in the URL otherwise it won't show.
for example, if the developer hits this URL -> local.magento24.com template hints won't show.
if developer pass param to URL like this local.magento24.com?templatehints=magento template hints will show.